en/dronecan/README.md

```diff
@@ -0,0 +1,114 @@
+# DroneCAN
+
+[DroneCAN](https://dronecan.github.io/) is a open software communication protocol for flight controllers and other [CAN](../can/README.md) devices on a vehicle to communicate with each other.
+It supports many different sensors and actuators connected over CAN.
+Connecting peripherals over DroneCAN has many benefits, including simpler wiring, telemetry/feedback from servos and ESCs, wiring redundancy, and more reliable performance (especially over longer wires on large vehicles).
+Additionally, it allows the user to configure and update the firmware of all CAN-connected devices centrally through PX4.
+
+:::note
+PX4 requires an SD card to enable dynamic node allocation and for firmware update.
+The SD card is not used in flight.
+:::
+
+:::note
+DroneCAN was previously known as UAVCAN v0 (or just UAVCAN).
+The name was changed in 2022.
+:::
+
+## Support
+
+### Supported Hardware
+
+Most common types of peripherals (sensors, ESCs, and servos) that are DroneCAN/UAVCAN v0 compliant are supported.
+
+Supported hardware includes (this is not an exhaustive list):
+
+- [ESC/Motor controllers](../dronecan/escs.md)
+- Airspeed sensors
+  - [Thiemar airspeed sensor](https://github.com/thiemar/airspeed)
+- GNSS receivers for GPS and GLONASS
+  - [ARK GPS](../dronecan/ark_gps.md)
+  - [ARK RTK GPS](../dronecan/ark_rtk_gps.md)
+  - [CubePilot Here3](https://www.cubepilot.org/#/here/here3)
+  - [Zubax GNSS](https://zubax.com/products/gnss_2)
+  - [CUAV NEO v2 Pro GNSS](https://doc.cuav.net/gps/neo-series-gnss/en/neo-v2-pro.html)
+  - [CUAV NEO 3 Pro GNSS](https://doc.cuav.net/gps/neo-series-gnss/en/neo-v2-pro.html)
+  - [CUAV C-RTK2 PPK/RTK GNSS](../gps_compass/rtk_gps_cuav_c-rtk2.md)
+- Power monitors
+  - [Pomegranate Systems Power Module](../dronecan/pomegranate_systems_pm.md)
+  - [CUAV CAN PMU Power Module](../dronecan/cuav_can_pmu.md)
+- Distance sensors
+  - [ARK Flow](ark_flow.md)
+  - [Avionics Anonymous Laser Altimeter UAVCAN Interface](../dronecan/avanon_laser_interface.md)
+- Optical Flow
+  - [Ark Flow](ark_flow.md)
+- Generic CAN Node (enables use of I2C, SPI, UART sensors on the CAN bus).
+  - [ARK CANnode](../dronecan/ark_cannode.md)
+
+## Hardware Setup
+
+DroneCAN operates over CAN. Refer to the [CAN setup documentation](../can/README.md) before continuing with this article.
```

Comment:
Probably best to make the link go to the specific wiring section. Also updating text to my taste.

```suggestion
DroneCAN operates over a CAN network.
DroneCAN hardware should be connected as described in [CAN > Wiring](../can/README.md#wiring).
```